### ISSUE TITLE
Extend 'famous_for' Property to have categories and add/adjust methods

### ISSUE THREAD

**Issue by sthiepaan** [bhatvikrant/world-countries-capitals#43]
As mentioned in #32 we decided to extend this package with:
- Categories for `famous_for` Property (thanks @jav7zaid for an idea!)
- All countries should contain all categories for that property (even if they don't have a value)
- Initial categories:
  - `building`
  - `food`
  - `sport`
- Each category is an `string[]` (array of strings)
- If category has no values it should be as as empty array
- For the project needs all properties (and values) have to be **lowercase**.

After refactoring, each `Country {Object}` should contain:
```js
{
  "country": "poland",
  "capital": "warsaw",
  "currency": "zloty",
  "native_language": ["polish"],
  "phone_code": "+48",
  "flag": "https://flagpedia.net/data/flags/h80/pl.png",
  "drive_direction": "right",
  // REFACTORED DATA
  "famous_for": {
    "building": ["palace of culture and science", "wawel"],
    "food": ["pierogi", "potatoes"],
    "sport": ["ski jumping", "volleyball"],
    // ...
  }
}
```

After updating/adding data, then we need to refacator `getCountriesByFamousFor(famousThing)` Method to have two parameters ➡️ `getCountriesByFamousFor(type, value)` (e.g.: `getCountriesByFamousFor("sport", "football")`).

**Comment by jav7zaid** [comment]
@sthiepaan are you working on this?

**Comment by sthiepaan** [comment]
No, just create a separate issues, so we do not flood a single thread. You can grab it!

@bhatvikrant if all is clear and fine to you, could you assign @jav7zaid? 😉
